packages/storybook/stories/VideoGallery/VideoGallery.stories.tsx

```diff
@@ -78,16 +80,99 @@ const getDocs: () => JSX.Element = () => {
         will be prioritized. Furthermore, the VideoGallery is designed to limit the re-ordering when the
         `dominantSpeakers` prop is changed.
       </Description>
-
-      <Heading>Horizontal Gallery</Heading>
+      <Heading>Overflow Gallery</Heading>
+      <DetailedBetaBanner></DetailedBetaBanner>
+      <Description>
+        In our video gallery we will show participants who are not to be prioritized in the grid view of the video
```

Comment:
Maybe this is better for this paragraph:
"In the video gallery, participants who are not prioritized in the grid view are placed in an overflow gallery. The overflow gallery can be placed on the bottom of the VideoGallery displaying participants horizontally by assigning the `overflowGalleryLayout` to 'HorizontalBottom'. This is the default. Conversely, the overflow gallery can placed on the right displaying participants vertically by assigning the `overflowGalleryLayout` to 'VerticalRight'."